### DISCUSSION TITLE
Why there is no wideopen documentation on how to install the library for Vue 2 ?

### DISCUSSION THREAD

**Discussion by klimentpetkov** [gilbitron/laravel-vue-pagination#135] [Help & support] [answered]
I see only two archives where the Vue 2 version resides but there is no ReadMe, on how to install it step by step

**Comment by epcgrs** [comment, marked as answer]
I think you can follow the v2 steps,
https://github.com/gilbitron/laravel-vue-pagination/tree/v2
